### ISSUE TITLE
PhysicsProcessor ParentScene may be initialized to null

### ISSUE THREAD

**Issue by manio143** [stride3d/stride#1021]
**Release Type**: GitHub

**Version**: master

**Platform(s)**: N/A

**Describe the bug**

I'm not currently sure why or if always but on the below line the `parentScene` is not set, which results in later inability to display debug colliders in game.
https://github.com/stride3d/stride/blob/74b2b4ad07d7811688e540259cedb8abb95ead9f/sources/engine/Stride.Physics/Engine/PhysicsProcessor.cs#L215

**To Reproduce**
1. Create a new project
2. Add a static collider to the ground
3. Add Debug physics shapes script
4. Launch game and press Ctrl+Shift+P

**Expected behavior**
Debug collider shapes are displayed.

**Additional context**
The issue can be easily mitigated by adding 
```csharp
SceneSystem.SceneInstance.GetProcessor<PhysicsProcessor>().ParentScene = Entity.Scene;
```
to the beginning of the debug script.

**Comment by kizilsum** [comment]
Hi! Is this still an issue? If so, I would like to take this one. @manio143 @Eideren

**Comment by Eideren** [comment]
@kizilsum Sure, go ahead !
Just to clarify, the `help wanted` message removed above is because anyone can now freely contribute to the engine. You can open PRs for any issue you want, right now the engine doesn't have a lot of traffic so you can just do so without notifying us first if you don't want to :)

**Comment by wrshield** [comment]
Hello, I'd like to try working on this issue and was wondering if anyone had some advice or ideas on where to start.  Thanks!

**Comment by Doprez** [comment]
https://doc.stride3d.net/4.2/en/manual/engine/entity-component-system/index.html I assume this will be an issue within the Entity processor? I havent messed with Bullet too much but each of these physics shapes should be getting initialized in the EntityProcessor if it works like other systems.

I assume it should be around [here](https://github.com/stride3d/stride/blob/55bd9529def5c4e2e5daa09a7d7e570745b6968c/sources/engine/Stride.Physics/Engine/PhysicsProcessor.cs#L134) where it gets set up.

**Comment by Eideren** [comment]
Fixed through #2262
